mitodl/hugo-course-publisher
# short name for left nav

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/hugo-course-publisher/issues/152

#### What's this PR do?
Updates sample markdown with the output of https://github.com/mitodl/ocw-to-hugo/pull/51 which sets the `short_page_title` property from the master json as the menu item name.

#### How should this be manually tested?
Run the site or visit the deploy preview to see the changes:

http://localhost:3000/courses/18-065-matrix-methods-in-data-analysis-signal-processing-and-machine-learning-spring-2018/sections/final-project/

vs

https://master--ocw-next.netlify.app/courses/18-065-matrix-methods-in-data-analysis-signal-processing-and-machine-learning-spring-2018/sections/final-project/

#### Screenshots (if appropriate)
![Screen Shot 2020-07-17 at 4 05 31 PM](https://user-images.githubusercontent.com/12089658/87826929-c31e5680-c847-11ea-89c7-396b981e56f7.png)
![Screen Shot 2020-07-17 at 4 06 21 PM](https://user-images.githubusercontent.com/12089658/87826932-c580b080-c847-11ea-8c92-f36048358613.png)
